Apps/Sandcastle/gallery/Video.html

```diff
@@ -33,7 +33,7 @@
 <div id="loadingOverlay"><h1>Loading...</h1></div>
 <div id="toolbar"></div>
 
-<video id="trailer" style="display: none;" autoplay loop crossorigin controls>
+<video id="trailer" style="display: none;" muted autoplay loop crossorigin controls>
```

Comment:
On my machine, the video wouldn't display until I actually showed the overlay, so let's remove `display:none` here and default the overlay to being shown for the best compatibility.

Reply:
That makes sense, I'll set that up